### ISSUE TITLE
Investigate MARS-E 2.0 Compliance

### ISSUE THREAD

**Issue by vrajmohan** [18F/eligibility-rules-service#157]
If we are hosting the registration app, we need to investigate MARS-E 2.0 Compliance for PII Security and DSNAP registration data transfer.

We obviously need to understand this for estimating Phase 4 work, but I'm assuming that we don't have to be compliant in Phase 3 as we will be working only with fake data. @edmullen?

**Comment by vrajmohan** [comment]
[Catalog of Minimum Acceptable Risk Security and Privacy Controls for Exchanges](https://drive.google.com/file/d/0B7B4Cxnx7uf2SWUxbWNMTWQ3UV9PdWRaZXU2bTA5ZFBEMEY4/view)

**Comment by edmullen** [comment]
No, we certainly are not required now or ever to meet MARS-E as it is specifically for the Health Insurance Exchanges. MARS-E is more reference material. But understanding the landscape that state agencies are familiar with and knowing what controls we'd need in place for full production will come at some point. Unclear when exactly. 

See also [appendix a in volume 2](https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/2-MARS-E-v2-0-Minimum-Acceptable-Risk-Standards-for-Exchanges-11102015.pdf) for crosswalk against nist 800-53 moderate baseline.

**Comment by edmullen** [comment]
We'll need to understand the compliance requirements on the fed side for both rules service and reg app as well as the state requirements for use of both.
